client/js/widgets/widget.js

```diff
@@ -942,6 +942,24 @@ export class Widget extends StateManaged {
         }
         if((a.property == 'parent' || a.property == 'deck') && a.value !== null && !widgets.has(a.value)) {
           problems.push(`Tried setting ${a.property} to ${a.value} which doesn't exist.`);
+        } else if (a.property == 'id' && isValidCollection(a.collection)) {
+          if (collections[a.collection].length) {
```

Comment:
If the operation is `+` then it is conceivable that this would work on multiple widgets at once. I've created a PR (#572) to change this.

Reply:
> If the operation is `+` then it is conceivable that this would work on multiple widgets at once. I've created a PR (#572) to change this.

I did consider this. Perhaps it's reasonable to keep this in a separate PR so we can think about it separately.

Reply:
It's also a bit of an odd use case.

Reply:
> I did consider this. Perhaps it's reasonable to keep this in a separate PR so we can think about it separately.

That's why I didn't commit to your branch.